modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,395 @@
+
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-imageset-config-parameters-v2_{context}"]
+= ImageSet configuration parameters for v2
+
+The oc-mirror plugin requires an image set configuration file that defines what images to mirror. The following table lists the available parameters for the `ImageSetConfiguration` resource.
+
+.`ImageSetConfiguration` parameters
+[cols="2,2a,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `ImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v2alpha1`.
+
+|`archiveSize`
+|The maximum size, in GiB, of each archive file within the image set.
+|Integer. For example: `4`
+
+|`mirror`
+|The configuration of the image set.
+|Object
+
+|`mirror.additionalImages`
+|The additional images configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml]
+----
+additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+----
+
+|`mirror.additionalImages.name`
+|The tag or digest of the image to mirror.
+|String. For example: `registry.redhat.io/ubi8/ubi:latest`
+
+|`mirror.blockedImages`
+|The full tag, digest, or pattern of images to block from mirroring.
```

Comment:
@sherine-k should we say here that images with manifest unknown errors only could be blocked  and not everything ?